wiki/en/FAQ.md

```diff
@@ -28,7 +28,7 @@ Bear in mind also that all participants should follow [The Golden Rule](/wiki/Cl
 
 ### How do I know if I can join a Server? Are there rules?
 
-In general, if somebody lists a Server on one of the Public Directories provided by Jamulus by default, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms built in. However, some Servers may state their policies in the welcome message you will see in the chat window.
+In general, if somebody lists a Server on a Directory, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms built in. However, some Servers may state their policies in the welcome message you will see in the chat window.
```

Comment:
Here "Public" is correct.  We only assert this for those built in directories.

For Custom Directories, it's up to the owner of the Directory to set whatever guidelines they'd like to announce and potentially enforce by whitelisting servers that can register, firewalling those servers, etc.

However, the whole para could be rewritten:
```
If somebody lists a Server on one of the Public Directories built into Jamulus, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms.  Whilst some Servers may state their policies in the welcome message you will see in the chat window, Jamulus itself does nothing to enforce these.
```

Reply:
The use of the term "Public" implies there are also "private" Directories though. But we don't want to use "Private", since it also implies some kind of authentication might be possible. 

So if we avoided the term "Public" by using "built-in" instead, I think that would make it easier?

`If somebody lists a Server on one of the Directories built into Jamulus, ...`

BTW note also the paragraph after it, which may further clarify the public/private thing too.

Reply:
It's a balance.  I'm very keen to keep the message that the built in Directories are for _Public_ use and that any Server registering with one is _inviting_ public usage.  So by calling them _Public_, that message is re-inforced.

Reply:
> BTW note also the paragraph after it, which may further clarify the public/private thing too.

No, that just says you don't have to register _at all_.

Reply:
There are Directories with different usage policies, such as private.jamulus.io which is intended to allow Servers to register for private use.  I also run a number of other directories, listed on jamulus.drealm.info, for people who want to register but for soem reason don't want to use the Public Directories.